build.gradle

```diff
@@ -33,7 +33,7 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.9.2'
+        classpath 'com.google.protobuf:protobuf-gradle-plugin:0.9.4'
```

Comment:
Is this change necessary for this PR?

Reply:
Yes otherwise kotlin-android doesnt work well with protobuf